pgcli/pgexecute.py

```diff
@@ -106,31 +108,35 @@ def run(self, sql):
             except:
                 _logger.debug('Database name missing.')
                 raise RuntimeError('Database name missing.')
-            self.conn = psycopg2.connect(database=dbname,
-                    user=self.user, password=self.password, host=self.host,
-                    port=self.port)
             self.dbname = dbname
-            self.conn.autocommit = True
+            self.connect()
```

Comment:
oof, just realized that this might break everything if connecting to the db failed. Fixing tomorrow.